### ISSUE TITLE
Repair JSON should remove trailing commas

### ISSUE THREAD

**Issue by imgx64** [josdejong/jsoneditor#610]
The Repair JSON button should remove trailing commas from objects and arrays. For example:
```
{
  "array": [
    1,
    2,
    3,
  ],
  "boolean": true,
}
```
is a valid JavaScript literal (but not valid JSON). So the Repair JSON button should change it to:
```
{
  "array": [
    1,
    2,
    3
  ],
  "boolean": true
}
```
which is the equivalent valid JSON.

**Comment by josdejong** [comment]
Great idea! Thanks for your suggestion

**Comment by josdejong** [comment]
Removing trailing commas is available now in `v5.26.3`.
